example/arq_prometheus_worker/worker.py

```diff
@@ -35,16 +33,15 @@ async def main():
         await redis.enqueue_job("download_content", url)
 
 
-# WorkerSettings defines the settings to use when creating the work,
+# WorkerSettings defines the settings to use when creating the worker,
 # it's used by the arq cli.
 # For a list of available settings,
 # see https://arq-docs.helpmanual.io/#arq.worker.Worker
-class WorkerSettings:
-    functions = [download_content]
-    on_startup = startup
-    on_shutdown = shutdown
-    health_check_interval = delay.total_seconds()
+class CustomWorker(Worker):
```

Comment:
Why change the worker? we now have one more import 🤔  and the way it was is the way is recommended in the [sample](https://arq-docs.helpmanual.io/#simple-usage) in the official docs, isn't it?